soc/riscv/riscv-privilege/telink_b91/start.S

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2021 Telink Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define NDS_MCACHE_CTL               0x7CA
+#define NDS_MMISC_CTL                0x7D0
+
+#include <toolchain.h>
+
+	.option push
+	.option norelax
+	.org 0x0
+
+/* exports */
+GTEXT(entry)
+
+SECTION_FUNC(init, init)
+entry:
+
+	j start
+
+	.org 0x20
+	.word ('T'<<24 | 'L'<<16 | 'N'<<8 | 'K')
+
+	.org 0x26
+	.short (0x173B)
+
+	.option pop
+	.align 2
+
+start:
+
+#if 0
+	/* Debug LED */
+	lui t0,0x80140
+	li t1, 0x0f
+	li t2, 0x10
+	sb t1 , 0x30a(t0)
+	sb t2 , 0x30b(t0)
+#endif
+
+#ifdef __nds_execit
+	/* Initialize EXECIT table */
+	la t0, _ITB_BASE_
+	csrw uitb, t0
+#endif
```

Comment:
How is `__nds_execit` defined here?

Reply:
NDS toolchain specific stuff.
We do not need it now. Removed.